packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -998,6 +998,8 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             { id: 'list.hoverBackground', defaults: { dark: '#2A2D2E', light: '#F0F0F0' }, description: 'List/Tree background when hovering over items using the mouse.' },
             { id: 'list.hoverForeground', description: 'List/Tree foreground when hovering over items using the mouse.' },
             { id: 'list.filterMatchBackground', defaults: { dark: 'editor.findMatchHighlightBackground', light: 'editor.findMatchHighlightBackground' }, description: 'Background color of the filtered match.' },
+            { id: 'tree.indentGuidesStrokeActive', defaults: { dark: '#585858', light: '#a9a9a9' }, description: "Tree Widget's stroke color for active indent guides." },
```

Comment:
Is it colors from VS Code? Did you test all color and icon themes?

Reply:
Yes. The colors are used by vscode. I have tested all colors and icon theme.

Reply:
Yes. The colors are used by vscode. I have tested all colors and icon theme.